src/shared/components/DynamicPageComponent/DynamicPageComponent.scss

```diff
@@ -68,3 +68,7 @@
     gap: 12px;
   }
 }
+
+.DynamicPage-contentContainer-0-2-26 {
+  padding-top: 0 !important;
+}
```

Comment:
idk if all the time it will be `-0-2-26` maybe better option will be to make it as `DynamicPage::part(content)` if it's possible

Reply:
I found the solution, in `DynamicPageHeader.js` it should be like this:
```
      headerContent={
        title !== 'Clusters Overview' ? (
          <DynamicPageHeader className="header-wrapper">
            {description && <Text className="description">{description}</Text>}
            <section
              className={`column-wrapper ${columnWrapperClassName || ''}`}
            >
              {children}
            </section>
          </DynamicPageHeader>
        ) : null
      }
```